lib/control-connection.js

```diff
@@ -25,10 +25,11 @@ var schemaChangeTypes = {
 /**
  * Represents a connection used by the driver to receive events and to check the status of the cluster
  * @param {Object} options
+ * @param {ProfileManager} profileManager
  * @extends EventEmitter
  * @constructor
  */
-function ControlConnection(options) {
+function ControlConnection(options, profileManager) {
```

Comment:
I agree that using the one from the defaultProfile makes the most sense (especially since it will usually be options.policies anyways, and in the case it isn't its probably because it was set explicitly in the default profile instead of options.policies)